rest/src/main/scala/io/udash/rest/util/Utils.scala

```diff
@@ -0,0 +1,16 @@
+package io.udash
+package rest.util
+
+import monix.eval.Task
+import monix.reactive.Observable
+
+import java.io.ByteArrayOutputStream
+
+private[rest] object Utils {
+
+  def mergeArrays(data: Observable[Array[Byte]]): Task[Array[Byte]] =
+    data.foldLeftL(new ByteArrayOutputStream()) { case (acc, elem) =>
+      acc.write(elem)
+      acc
+    }.map(_.toByteArray)
+}
```

Comment:
nit: I'd rather not ship this powerful "Utils" at all, but if we have to is it in any way faster than `data.flatMap(Observable.fromIterable(_)).toL(Array)` that could be inlined? if, so needs a comment and a benchmark